src/Files.App/Helpers/Layout/LayoutPreferencesManager.cs

```diff
@@ -265,6 +265,24 @@ public Type GetLayoutType(string path, bool changeLayoutMode = true)
 			};
 		}
 
+		public void UpdateGroupAndSortOptions(string? path)
+		{
+			if (string.IsNullOrWhiteSpace(path))
+				return;
+
+			var preferencesItem = GetLayoutPreferencesForPath(path);
```

Comment:
Do we need to reload the preferences, or can we simplify this further by just invoking the related events (eg `GroupDirectionPreferenceUpdated`)?

Reply:
We cannot just invoke the event because the properties are not updated unless we reload them. I can add a parameter that tells which is the only one to update

Reply:
BTW should we sync sorting preferences even when they are set clicking on the details view headers? If so I'll implement it

Reply:
> We cannot just invoke the event because the properties are not updated unless we reload them.

I see, in that case I would just leave it as is.

> BTW should we sync sorting preferences even when they are set clicking on the details view headers? If so I'll implement it

It might be best to wrap up this PR first, but we can look into this next.